doc/users/prev_whats_new/github_stats_3.0.0.rst

```diff
@@ -926,13 +926,13 @@ Pull Requests (598):
 * :ghpull:`10955`: Py3fy cbook, compare_backend_driver_results
 * :ghpull:`10680`: Rewrite the tk C blitting code
 * :ghpull:`9498`: Move title up if x-axis is on the top of the figure
-* :ghpull:`10942`: Make active param in CheckBottons optional, default false
+* :ghpull:`10942`: Make active param in CheckButtons optional, default false
```

Comment:
I'd leave all `github_stats*` files as is. IMHO having the title exactly as in github (=consistency) is more important than having it spelled correctly.